src/Features/LanguageServer/ProtocolUnitTests/Completion/CompletionTests.cs

```diff
@@ -1362,6 +1377,447 @@ void M()
             Assert.Empty(results.Items);
         }
 
+        [Fact]
+        public async Task TestWhitespaceSettingsNameInEditorconfigFiles()
+        {
+            var markup = @"
+<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"" FilePath=""z:\TestProject\Assembly1.csproj"">
+        <Document FilePath=""File1.cs"">
+class C { }
+        </Document>
+        <AnalyzerConfigDocument FilePath=""z:\TestProject\.editorconfig"">
+indent_siz{|caret:|}
```

Comment:
can we add one test where nothing is typed on the line to verify we show the keys?